content/en/docs/concepts/configuration/pod-priority-preemption.md

```diff
@@ -145,6 +151,47 @@ globalDefault: false
 description: "This priority class should be used for XYZ service pods only."
 ```
 
+### Non-preempting PriorityClasses (alpha)
```

Comment:
```suggestion
### Non-preempting PriorityClasses (alpha) {#non-preempting-priority-class}
```

I would take the feature status out of the ID attribute so that the URI for this section stays stable as the feature graduates.